go.mod

```diff
@@ -6,13 +6,13 @@ require (
 	github.com/NYTimes/gziphandler v1.1.1
 	github.com/antchfx/jsonquery v1.1.5
 	github.com/bufbuild/buf v1.3.1
-	github.com/conduitio/conduit-connector-file v0.0.0-20220304165706-4cd6c275d2b0
-	github.com/conduitio/conduit-connector-generator v0.0.0-20220307173009-a7744489e9db
-	github.com/conduitio/conduit-connector-kafka v0.0.0-20220329121707-8ec203838b8c
-	github.com/conduitio/conduit-connector-postgres v0.0.0-20220309140906-3330671a8955
-	github.com/conduitio/conduit-connector-protocol v0.0.0-20220304172523-dcb690811ba6
-	github.com/conduitio/conduit-connector-s3 v0.0.0-20220308112711-bb3e0119a762
-	github.com/conduitio/conduit-connector-sdk v0.0.0-20220404170930-453c4ad1fafc
+	github.com/conduitio/conduit-connector-file v0.1.0
+	github.com/conduitio/conduit-connector-generator v0.1.0
+	github.com/conduitio/conduit-connector-kafka v0.1.0
+	github.com/conduitio/conduit-connector-postgres v0.1.0
+	github.com/conduitio/conduit-connector-protocol v0.2.0
+	github.com/conduitio/conduit-connector-s3 v0.1.1
```

Comment:
v0.1.1 and v0.1.0 point to the same commit.

Reply:
Yeah they do, but go getting v0.1.0 doesn't work because I already released it on march 1st and go proxy now doesn't want to overwrite it, see:
- https://proxy.golang.org/github.com/conduitio/conduit-connector-s3/@v/v0.1.0.info
- https://proxy.golang.org/github.com/conduitio/conduit-connector-s3/@v/v0.1.1.info

Reply:
The workaround was to create `v0.1.1` just for the sake of having a fresh version.